packages/sqlite/PKGBUILD

```diff
@@ -12,7 +12,7 @@ url='https://sqlite.org/'
 license=(PublicDomain)
 groups=()
 depends=()
-makedepends=(readline-dev)
+makedepends=(readline-dev 'ld-musl-x86_64.so.1')
```

Comment:
Nice, thanks!

Looking forward, I'd like to make this available on other architectures, at the very least, arm64. So we should probably either use "ld-musl-$(arch).so.1" or "ld-musl-${CARCH}.so.1". Either one should evaluate to what you have here.

Reply:
Here is what I've been able to build till now.
https://termbin.com/fewx

I'll try making the changes you propose